Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -79,6 +79,20 @@ public float RadialRange
             set { radialRange = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Distance for plane layout")]
```

Comment:
nit: specify units (presuming meters)

Reply:
none of the other fields specify it's units. You sure that people wouldn't just assume its meters, like everything else is? Can't remember anything being something else actually.

Reply:
Default is meters, which I think is fairly common enough that not specifying here is fine.

I think the times where it does become value is time (i.e. is it seconds, milliseconds, etc)